code/modules/mob/living/carbon/human/species.dm

```diff
@@ -879,17 +879,22 @@
 
 /datum/species/proc/movement_delay(mob/living/carbon/human/H)
 	. = 0
+	var/obj/item/device/flightpack/F = H.get_flightpack()
```

Comment:
er you mean add one specifically for flightpacks because this is intentional to be separate from wings